### PR TITLE
Add golang 1.6 and 1.7 images

### PR DESCRIPTION
**Pull request by dharmit**
ping @bamachrn @mohammedzee1000 

This is with regards to adding language/framework images to r.c.o

### CODE REVIEW

**Comment by mohammedzee1000** [comment]
@dharmit you will need to rebase this

**Comment by dharmit** [comment]
Rebased

File: index.d/dharmit.yml [hunk 1]
```diff
@@ -44,3 +43,24 @@ Projects:
     notify-email: shahdharmit@gmail.com
     depends-on: centos/centos:latest
 
+ - id: 7
```
Comment (mohammedzee1000) [inline index.d/dharmit.yml]: @dharmit The spacing here does not match with rest of file.

Comment (dharmit) [inline index.d/dharmit.yml]: Pushed the fix.

**Pull request merged**